packages/where/index.js

```diff
@@ -9,7 +9,7 @@ const { raw } = String;
 /**
  * @param {{[name: string]: string}} env
```

Comment:
I wonder if this is consistent with `typeof process.env`, which, in node, seems to be `Record<string, string|undefined>`.